website/docs/r/api_management_api.html.markdown

```diff
@@ -0,0 +1,124 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_api"
+sidebar_current: "docs-azurerm-resource-api-management-api-x"
+description: |-
+  Create a API Management API.
+---
+
+# azurerm_api_management_api
+
+Create a API Management API component.
+
+## Example Usage (import from Open API spec)
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "api-rg-dev"
+  location = "West Europe"
+}
+
+resource "azurerm_api_management" "test" {
+  name                = "api-mngmnt-dev"
+  publisher_name      = "My Company"
+  publisher_email     = "company@terraform.io"
+  sku {
+    name     = "Developer"
+    capacity = 1
+  }
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_api_management_api" "test" {
+  name                = "conferenceapi"
+  service_name        = "${azurerm_api_management.test.name}"
+  path                = "conference"
+  import {
+    content_format = "swagger-link-json"
+    content_value  = "http://conferenceapi.azurewebsites.net/?format=json"
+  }
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the API Management API.
+
+* `resource_group_name` - (Required) The Name of the Resource Group where the API Management API exists.
+
+* `location` - (Required) The Azure location where the API Management API exists.
+
+* `service_name` - (Required) The Name of the API Management Service where the API Management API exists
```

Comment:
since this is ForceNew, can we suffix this with: `Changing this forces a new resource to be created.`